src/fparser/two/tests/test_Fortran2003.py

```diff
@@ -106,2188 +105,2150 @@ def test_Program():  # R201
       subroutine foo (*)
       end subroutine foo
       ''')
-    obj = cls(reader)
-    assert isinstance(obj, cls), repr(obj)
+    obj = tcls(reader)
+    assert isinstance(obj, tcls), repr(obj)
     assert 'SUBROUTINE foo(*)\nEND SUBROUTINE foo' in str(obj)
 
 
-def test_Specification_Part():  # R204
-    ''' Tests for parsing specification-part '''
+def test_specification_part():
+    ''' Tests for parsing specification-part (R204) '''
     reader = get_reader('''\
     integer a''')
-    cls = Specification_Part
-    obj = cls(reader)
-    assert isinstance(obj, cls), repr(obj)
+    tcls = Specification_Part
+    obj = tcls(reader)
+    assert isinstance(obj, tcls), repr(obj)
     assert str(obj) == 'INTEGER :: a'
     assert (repr(obj) == "Specification_Part(Type_Declaration_Stmt("
             "Intrinsic_Type_Spec('INTEGER', None), None, "
             "Entity_Decl(Name('a'), None, None, None)))")
 
-    obj = cls(get_reader('''\
+    obj = tcls(get_reader('''\
 type a
 end type a
 type b
 end type b
 '''))
-    assert isinstance(obj, cls), repr(obj)
+    assert isinstance(obj, tcls), repr(obj)
     assert 'TYPE :: a\nEND TYPE a\nTYPE :: b\nEND TYPE b' in str(obj)
 
 #
 # SECTION  3
 #
 
 
-def test_Name():  # R304
+def test_name():  # R304
 
-    a = Name('a')
-    assert isinstance(a, Name), repr(a)
-    a = Name('a2')
-    assert isinstance(a, Name), repr(a)
-    a = Designator('a')
-    assert isinstance(a, Name), repr(a)
-    a = Constant('a')
-    assert isinstance(a, Name), repr(a)
-    a = Expr('a')
-    assert isinstance(a, Name), repr(a)
+    obj = Name('a')
+    assert isinstance(obj, Name), repr(obj)
+    obj = Name('a2')
+    assert isinstance(obj, Name), repr(obj)
+    objd = Designator('a')
+    assert isinstance(objd, Name), repr(objd)
+    objc = Constant('a')
+    assert isinstance(objc, Name), repr(objc)
+    obje = Expr('a')
+    assert isinstance(obje, Name), repr(obje)
 
 
-def test_Literal_Constant():  # R305
+def test_literal_constant():  # R305
 
-    cls = Constant
-    a = cls('.false.')
-    assert isinstance(a, Logical_Literal_Constant), repr(a)
-    assert str(a) == '.FALSE.'
+    tcls = Constant
+    obj = tcls('.false.')
+    assert isinstance(obj, Logical_Literal_Constant), repr(obj)
+    assert str(obj) == '.FALSE.'
 
 
-def test_Literal_Constant():  # R306
-
-    cls = Literal_Constant
```

Comment:
I don't think so because there are separate tests for different types of literal constants (e.g. `test_logical_literal_constant()`, `test_int_literal_constant()`. I think a type of constant was just randomly selected for this test, so I did not change anything. I'm happy to add more info if required.

Reply:
I meant that we now have no explicit test for the `Literal_Constant` class (R305) and probably we should ;-)